controllers/blog.js

```diff
@@ -0,0 +1,65 @@
+const articleModel = require('../models/blog')
+
+const createNewArticle = (req, res) => {
+  const newBlog = new articleModel({
```

Comment:
Can you use content from the frontend part? You can access that in req.body
And in postman send a body with a request that holds the info. Instead of hardcoding the content.

Reply:
thanks!
will work on that!